<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ProSolar CTPI Server</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnIniciarTD">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 170, 255);
	color: rgb(255, 255, 255);
	font: 100 12pt &quot;Verdana&quot;;
	border: 2px solid transparent;
	border-radius: 5px;
	padding:2px;
}

QPushButton:pressed
{
	background-color: rgb(60, 200, 255) !important;
}</string>
    </property>
    <property name="text">
     <string>Iniciar toma de datos</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtConsole">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>700</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(0, 85, 255);
font: italic 14pt &quot;Verdana&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDetenerTD">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 30, 30);
	color: rgb(255, 255, 255);
	font: 100 12pt &quot;Verdana&quot;;
	border: 2px solid transparent;
	border-radius: 5px;
	padding:2px;
}

QPushButton:pressed
{
	background-color: rgb(220, 60, 60) !important;
}</string>
    </property>
    <property name="text">
     <string>Detener toma de datos</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
